### ISSUE TITLE
How to auto import custom directive in Vue 2?

### ISSUE THREAD

**Issue by velmer** [antfu/unplugin-vue-components#454]
The title is self-explanatory, is there a way to do this?

```
src    
│
└───components
│   │   Example.vue
│   
└───directives
    │   color.ts
```

Example.vue
```vue
<template>
  <div v-color="'blue'">Example</div>
</template>
```

color.ts
```ts
export default (el, binding) => {
  el.style.color = binding.value;
};
```

I also tried with named export: `export const color = (el, binding) => ...`

I set the `directives` flag to true and installed Babel, but it didn't work. Is it just missing configuration on my end or there is no way to do this as of now?

PS: I don't know if here is the right place to ask questions, but I couldn't find a more appropriate one. If there is one, please let me know that I will move this question there.

**Comment by duncan-oxd** [comment]
Curious about this too. Do you get errors when you set `directives` to trueto true in the plugin config?  I'm trying to use Vuetify and it throws errors if I do that. The directives also don't work if I manually register them. I don't get why we would need to  use Babel here, so there's obviously something I'm missing.

**Comment by sxzz** [comment]
FYI: https://github.com/sxzz/vue2-directive-demo

**Comment by dews** [comment]
> FYI: https://github.com/sxzz/vue2-directive-demo
> 
> Try the latest version, fixed in #487

 I got 404 about you demo link
